src/js/app.js

```diff
@@ -21,6 +21,14 @@ import Lazzzy from './utils/lazzzy';
 	//       });
 	//   }
 	// }
+	const navigationToggleButton = document.querySelector('.nav__button');
+	const navigationWrapper = document.querySelector('.rss__nav');
+
+	navigationToggleButton.addEventListener('click', toggleNavigation);
```

Comment:
__[ESLint]__

Error - 'toggleNavigation' was used before it was defined. (no-use-before-define)


<!-- Comment by [SideCI](https://sideci.com) -->